Pod/Classes/Forms/UIView+Lookup.swift

```diff
@@ -16,41 +16,27 @@ extension UIView {
     /// - parameter includeNonInteractable: Return views that do not have user interaction enabled. The default is false.
     ///
     /// - returns: An array of views, of type T, sorted by y position.
-    public func lookupSortedViews<T: UIView>(includeHidden: Bool = false, includeNonInteractable: Bool = false) -> [T] {
-        if let textField = self as? T {
+    final public func sortedSubviews<T: UIView>(ofType type: T.Type, includeHidden: Bool = false, includeNonInteractable: Bool = false) -> [T] {
```

Comment:
#? Need prefixing? I'm still unclear on the rules regarding Swift, but I believe we want to `rz_` prefix this.